en/components/general/update_guide.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Update Guide
+_description: Information on updating to a newer version of the Ignite UI for Angular library.
+_keywords: Ignite UI for Angular, update, npm package, version, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library
+---
+
+## Ignite UI for Angular update guide
+
+From the table below you may select to what version of Ignite UI for Angular you want to update your project.
+
+-   [7.0 to latest](#70)
+-   [6.2 to latest](#70)
+-   [6.1 to latest](#61)
+-   [6.0 to latest](#61)
+
+#### <a id="70"></a>Updating Ignite UI for Angular from versions 6.2/7.0 to latest
```

Comment:
If the first one is heading 1, this will look better with heading 2. The same comment might be applied to the other section